bedrock/cms/wagtail_localize_smartling/callbacks.py

```diff
@@ -18,19 +18,24 @@
 
 if TYPE_CHECKING:
     from wagtail_localize_smartling.models import Job
-
+from sentry_sdk import capture_exception, capture_message
 from wagtaildraftsharing.models import WagtaildraftsharingLink
 from wagtaildraftsharing.views import SharingLinkView
 
 
 def _get_html_for_sharing_link(sharing_link: WagtaildraftsharingLink) -> str:
     request = RequestFactory().get(sharing_link.url)
     view_func = SharingLinkView.as_view()
-    resp = view_func(
-        request=request,
-        key=sharing_link.key,
-    )
-    return resp.content.decode("utf-8")
+    try:
+        resp = view_func(
+            request=request,
+            key=sharing_link.key,
+        )
+        return resp.content.decode("utf-8")
+    except Exception as ex:
```

Comment:
I know this is very broad, but it's important to cast a wide net for the sentry logging